xml/OptionPanel.xml

```diff
@@ -0,0 +1,123 @@
+<!-- Default formatting -->
+<Defaults>
+  <Text color="white" alignment="MiddleLeft"/>
+  <Toggle isOn="False" rectAlignment="MiddleRight"/>
+
+  <VerticalLayout class="window" active="false" color="black" visibility="Admin" outlineSize="1 1" outline="grey" allowDragging="true" returnToOriginalPositionWhenReleased="false" width="500" height="800"/>
+  <VerticalLayout class="group" outlineSize="1 1" outline="grey" color="#222222" padding="10" spacing="5"/>
+  <VerticalLayout class="text-column" padding="5 20 0 0"/>
+
+  <HorizontalLayout class="group-content" color="#444444" padding="5" spacing="5"/>
+
+  <Text class="group-header" fontSize="24" font="font_teutonic-arkham"/>
+  <Text class="option-header" fontSize="16" font="font_teutonic-arkham"/>
+  <Text class="description" fontSize="12"/>
+
+  <Button class="bottomButtons" hoverClass="hover" pressClass="press" selectClass="select" color="#888888" minHeight="35" fontSize="24" font="font_teutonic-arkham"/>
+  <Button class="hover" color="grey"/>
+  <Button class="press" color="white"/>
+  <Button class="select" color="white"/>
+</Defaults>
+
+<!-- Option Panel -->
+<VerticalLayout id="optionPanel" class="window">
+  <Panel minHeight="45" flexibleHeight="0" padding="10 10 0 0">
+    <Text font="font_teutonic-arkham" fontSize="35">Options</Text>
+  </Panel>
+  <VerticalLayout>
+    <!-- Group 1 -->
+    <VerticalLayout class="group">
+      <Panel minHeight="44" image="option_image1" padding="5 0 0 0">
+        <Text class="group-header">PLAYERMAT SETTINGS</Text>
+      </Panel>
+
+      <!-- Option 1 -->
+      <HorizontalLayout class="group-content">
+        <VerticalLayout class="text-column">
+          <Text class="option-header">Enable snap tags</Text>
+          <Text class="description">Only cards with the tag "Asset" will snap (official cards are supported by default).&#xA;Disable this if you are having issues with custom content.</Text>
+        </VerticalLayout>
+        <Toggle id="toggle1" onValueChanged="onClick_toggleOption(1)" />
```

Comment:
These really should use meaningful IDs rather than generic numbering.  We're not going to be able to swap them out at will as each button is hard-linked to a given option, so the generic IDs just obfuscate what's what.

Reply:
Assigning them numbered indexes makes saving/loading options way easier

Reply:
A lookup map sounds like a good solution!
Would like to defer that until part 2 has made it's way past the merge because this would require changes at a lot of different positions in the code.

Reply:
Fair enough.